backend/lib/resolve-payload-cids.js

```diff
@@ -10,6 +10,7 @@ import {
 // @ts-ignore
 } from 'index-provider-peer-id'
 import { rpcRequest } from './rpc-service/service.js'
+import { assert } from 'node:console'
```

Comment:
```suggestion
import { assert } from 'node:assert'
```

Reply:
More powerful to use https://nodejs.org/api/assert.html than https://nodejs.org/api/console.html#consoleassertvalue-message, we basically use that everywhere